server/middleware/validations/profile.js

```diff
@@ -0,0 +1,92 @@
+const { check, validationResult } = require('express-validator/check');
+
+const profileValidator = {
+  newProfileValidator: [
+    check('age')
+      .not().isEmpty()
+      .withMessage('Age is required')
+      .trim()
+      .isNumeric()
+      .withMessage('Enter a valid age'),
+    check('gender')
+      .not().isEmpty()
+      .withMessage('Gender is required')
+      .trim()
+      .isIn(['male', 'Male', 'female', 'Female'])
+      .withMessage('Enter a valid gender'),
+    check('phoneNumber')
+      .not().isEmpty()
+      .withMessage('Phone number is required')
+      .trim()
+      .isNumeric()
+      .withMessage('Enter a valid phone number')
+      .isLength({ min: 11, max: 11 })
```

Comment:
What if I have a 7 digit phone number or I need to add country code?

Reply:
I'd take this down too.